json-csharp/IteratingOverJsonObject/IteratingOverJsonObject/JsonIteration.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Text.Json;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace IteratingOverJsonObject;
+
+public class JsonIteration
+{
+    public string Json { get; set; }
+
+    public JsonIteration()
+    {
+        Json = new TestData().GenerateJsonData();
+    }
+
+    public int IterateOverJsonDynamically()
+    {
+        var jsonData = JsonConvert.DeserializeObject<dynamic>(Json);
+
+        foreach (var data in jsonData)
+        {
+            var name = data.name;
+            var age = data.age;
+            var department = data.department;
+
+            Console.WriteLine("Name: {0}, Age: {1}, Department: {2} \n", name, age, department);
+        }
+
+        var count = jsonData.Count;
+        return count;
+    }
+
+    public int IterateUsingJArray()
+    {
+        var jsonArray = JArray.Parse(Json);
+
+        foreach (var data in jsonArray)
+        {
+            var name = (string)data["name"];
+            var age = (int)data["age"];
+            var department = (string)data["department"];
+
+            Console.WriteLine("Name: {0}, Age: {1}, Department: {2} \n", name, age, department);
+        }
+
+        var count = jsonArray.Count;
+        return count;
+    }
+
+    public List<Employee> IterateUsingDynamicallyTypedObject()
```

Comment:
This should be named `IterateUsingStaticallyTypedObject`

Reply:
This is also done, please take a look